### DISCUSSION TITLE
Delete specific Importer tags via UI

### DISCUSSION THREAD

**Discussion by digitlength** [firefly-iii/firefly-iii#9212] [Firefly III] [answered]
Is there a way, using the UI, to easily delete the "Data Import on..." tags set by the importer, but not touch the other tags?

Classification -> Tags and then selecting each one by one doesn't count, because there are hundreds! Unless I've missed a "select all" option somewhere?

Couldn't find how to do it in rules either, unless there's a wildcard symbol I can use?

**Comment by JC5** [comment, marked as answer]
There is no "select all" I'm afraid, but try this.

Open the development console (Ctrl-Shift-i) and add the following text:

```javascript
var list  =$('a:contains("Data Import")');
for(var i=0;i<list.length;i++) {
   var current = list[i];
   var prev = $(list[i]).prev();
   prev.prop('checked', true);
};
```

That should select all the tags you want, and then you can use the mass delete button at the bottom.
